packages/core/src/docs/classes.md

```diff
@@ -4,7 +4,11 @@ tag: new
 
 @# Classes
 
-Blueprint packages provide React components in JS files and associated styles in a CSS file. Each package exports a `Classes` constants object in JavaScript that contains keys of the form `NAMED_CONSTANT` for every CSS class used. This separation allows us to change CSS classes between versions without breaking downstream users (although in practice this happens very rarely).
+Blueprint packages provide React components in JS files and associated styles in
+a CSS file. Each package exports a `Classes` constants object in JavaScript that
+contains keys of the form `NAMED_CONSTANT` for every CSS class used. This
+separation allows us to change CSS classes between versions without breaking
+downstream users (although in practice this happens very rarely).
```

Comment:
wrapped all these lines so they're more legible in editors. common practice in other files.